cropbox_filter_node:
  ros__parameters:
    input_cloud: "/velodyne_points"
    output_cloud: "/cropbox_filter/velodyne_points"
    min_x: -0.5
    min_y: -0.5
    min_z: -0.5
    max_x: 0.5
    max_y: 0.5
    max_z: 0.5
